src/System.Collections/src/System.Collections.csproj

```diff
@@ -20,6 +20,11 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard1.7_Release|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcoreapp1.1_Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcoreapp1.1_Release|AnyCPU'" />
+  <ItemGroup Condition="'$(TargetGroup)' == 'net463'">
+    <ContractProject Include="..\ref\System.Collections.csproj">
+      <TargetGroup>netstandard1.7</TargetGroup>
+    </ContractProject>
```

Comment:
I was confused by this usage, but I see this is new functionality for an older concept - https://github.com/dotnet/buildtools/commit/5da1a3e63c67e12ae5ab4073c9ac289e1713e86e

What does this do exactly?  Does it ensure that there is a project reference to the reference assembly project?  Are there scenarios where we actually have to define the OSGroup for these?